### ISSUE TITLE
Design: Create an empty state for our tables

### ISSUE THREAD

**Issue by jillian-hammer** [CDCgov/prime-reportstream#9461]
## User story

- As a ReportStream user, I need to understand that no data resulted from my filtering/searching so that I can move onto troubleshooting in other ways.
- As a ReportStream designer/engineer, I need a default empty state to apply to tables so that we can apply them easily across the site with less churn per use case.

## Background & context

This work is part of the custom table component MVP. The custom component will also allow for customized future features that will not only meet our users needs, but also standardize tables across our website and make implementing new tables more efficient.

## Open questions

-   Will the empty state be sufficient enough to serve all use cases of empty tables?
-   Is the solution UX copywriting, visual UI, or both?

## Related links
-   [Table Design Discovery Mural](https://app.mural.co/t/primedatainput6410/m/primedatainput6410/1677789052306/05f1e11a2448c3349735fe808833da01c6c0283b?sender=u0c3cdba5126734aceb613229) (see empty state section)

## Deliverable 
Figma [Table Styles Spec ](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?type=design&node-id=5773%3A40927&mode=design&t=FI0e5zlEmrIZvPYl-1) 

## Acceptance criteria
-   [x] Design an empty state that is flexible enough to meet the majority of use cases across the site
-   [x] Review with Engineering
-   [x] Update anything that needs to be updated/clarified

**Comment by yudonoodles** [comment]
Hi @etanb can you review the no results found translation and accompanying visual I created when you apply filters for tables that deliver no results? [Link](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?type=design&node-id=5749%3A40835&mode=design&t=0zt17gW7KgdLWC9m-1) to example screen. 

Also tagging @audreykwr to review the translation itself. 

cc: @jillian-hammer

**Comment by yudonoodles** [comment]
Etan and Audrey reviewed the Figma page with table specs and the empty table state, and provided comments directly on Figma.

**Comment by yudonoodles** [comment]
@etanb, thanks for your feedback. 

When you have a moment, can you jump back in the [file](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?type=design&node-id=5773%3A40927&mode=design&t=ggqyDtPbhmoP4VgO-1) and review my comment. 

Here's the main question we are waiting on a response for: 
Based on two scenarios, one where the filtering leads to no results and two, when data is not sent so table is empty/missing, can we show different sets of feedback to users? It'd be ideal to provide users context-driven feedback to make it clear what they need to do next.

**Comment by yudonoodles** [comment]
Reviewed feedback left by Etan and Audrey. 

Regarding the scenario posed above, Etan shared: "If it's not a deal breaker, having a generic message would slot into how the app works way better. We CAN determine through some convoluted logic the states you mention above, but if it's not a priority, a generic message would be great."

In response to that, Audrey shared that "My opinion would be that because the dashboard use may taken some time to ramp up, we can backlog more specific messages as roll out gets going."

Based on the two pieces of feedback, we are moving towards showing a generic message/feedback for users, unless prioritized in the future to account for different use cases (as mentioned above). 

I updated the design with the new message that Audrey recommended and tagged her to the design.

**Comment by yudonoodles** [comment]
Waiting on updated content from @audreykwr.

**Comment by yudonoodles** [comment]
Updated with new content. Ready to be closed.
